manifests/init.pp

```diff
@@ -63,16 +65,16 @@
 
   String  $admin_user = 'admin',
   Sensitive[String[1]] $admin_pass = Sensitive('puppetlabs'),
-  String  $token_file = $facts['identity']['user'] ? {
-                                      'root'  => '/root/.influxdb_token',
-                                      default => "/home/${facts['identity']['user']}/.influxdb_token"
-                                    },
-
-){
+  String  $token_file = "/home/${facts['identity']['user']}/.influxdb_token",
 
+) {
+  $real_token_file = $facts['identity']['user'] ? {
```

Comment:
Is this change just to remove the selector from the class parameter?  I'm not sure we should do this because it could create a discrepancy between the class param and this variable.

For example if I set `influxdb::token_file` to something other than the default, like `'/root/some_other_file'` and run the agent as `root`, then `$real_token_file` would ignore the class parameter and set it to `'/root/.influxdb_token'`